WooCommerce/Classes/ViewRelated/Orders/Order Creation/Synchronizer/OrderSynchronizer.swift

```diff
@@ -150,4 +155,8 @@ protocol OrderSynchronizer {
     /// Commits all order changes to the remote source. State needs to be in `.synced` to initiate work.
     ///
     func commitAllChanges(onCompletion: @escaping (Result<Order, Error>, _ usesGiftCard: Bool) -> Void)
+
+    /// Sets the block behavior for sync requests
+    ///
+    func updateBlockingBehavior(_ behavior: OrderSyncBlockBehavior)
```

Comment:
Is this new method something we should test through `RemoteOrderSynchronizerTests`? I see we only update it through forwarding the sync state in the `EditableOrderViewModel` so I'm not sure if is worth it, but we do have other tests to check the current sync blocking state when an order update is triggered 🤔. I thought just worth mentioning!

Reply:
Yep, I'll look at adding tests as long as they are worthwhile, good shout.